src/Server/middleware/userRequiredMiddleware.ts

```diff
@@ -4,7 +4,6 @@ const USER_REQUIRED_ROUTES = [
   "/notifications",
   "/orders(.*)",
   "/settings/alerts",
-  "/settings/my-collection",
```

Comment:
since this is not yet hidden behind a ff, we will have to be careful with the release of it